docs/_data/table/en/user-configuration.yml

```diff
@@ -30,6 +30,9 @@ alluxio.user.file.worker.client.threads:
   How many threads to use for file worker clients to read from workers.
 alluxio.user.file.write.location.policy.class:
   The default location policy for choosing workers for writing a file's blocks
+alluxio.user.file.write.capacity.reserved.ratio:
```

Comment:
Why is this property useful? We don't use anything like this for the regular `LocalFirstPolicy` so it seems strange to use it here. Is it possible to break adding this property into a separate PR so that it doesn't block merging the new location policy?